### ISSUE TITLE
Repo Name Change

### ISSUE THREAD

**Issue by HFTSOL** [Foxandxss/angular-toastr#225]
Given the rebranding of **Angular <= 1.5** to **AngularJS** and **Angular >= 2** to **Angular**, I suggest renaming the repo to something more inline with the new naming scheme.  I believe that would prevent much of the confusion that would be caused by this being named **angular-toastr** when it is for **AngularJS**.

**Comment by Foxandxss** [comment]
Yes, I actually need to do something like that. Problem is that the package still have that name.

Let's see if I can update a few needed things in code and then create a new version with the new name.
